### ISSUE TITLE
Error MKLiveCD

### ISSUE THREAD

**Issue by astronom1987** [unity-linux/mklivecd#9]
In the process of creating an image, this error appears 
`Creating compressed image:                         [100.00% 00:40:54/00:41:26]chmod: unable to access /tmp/mklivecd.4392/livecd/loopbacks/distrib-lzma.sqfs': There is no such file or directory=========================================== ]
DIE
FATAL: Execution of 'chmod 644 /tmp/mklivecd.4392/livecd/loopbacks/distrib-lzma.sqfs' failed`
There is really no such file, but it is not clear why. What does this program do if it cannot do something for what it was created: add packages to the image of lzma.sqfs?
Mageia 7.0 RC, XFCE. 
Sorry for my English, I use a translator

**Comment by jmiahman** [comment]
This project is no longer maintained, there's been a few changes to Mageia
that broke it's functionality and due to lack of interest was never
updated. Feel free to send in patches.

**Comment by astronom1987** [comment]
Thanks for the answer, I will know that nothing of this project for Mageia should be expected.

**Comment by jmiahman** [comment]
Yeah I have written something called GOLiveCD for Fedora based distros that
may work now for Mageia but it hasn't been tested. I attempted to be a part
of the Mageia community even got sponsored etc.. but they weren't a very
welcoming community and in some aspects were insulting, so I moved on. If
there's peaked interest for such a solution then I may add support in
GOLiveCD for Mageia, but without interest or testers there's no point in
considering development.
